### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by The-Man-Is-Me** [3kh0/ext-remover#125] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Just wanted to thank the coders behind this exploit and everyone that has helped make and improve it.

**Comment by ghost** [comment]
I know I haven't contributed to the exploit (yet), but you're very welcome

**Comment by 3kh0** [comment, marked as answer]
Please be sure to thank the original creator, bypassi! If you want to support me, make sure you star the repo!
